pom.xml

```diff
@@ -2140,7 +2140,7 @@
             <dependency>
                 <groupId>io.projectreactor.rabbitmq</groupId>
                 <artifactId>reactor-rabbitmq</artifactId>
-                <version>1.0.0.RELEASE</version>
+                <version>1.2.0.RELEASE</version>
```

Comment:
you should upgrade reactor at the same time (to Californium-SR7 probably) because reactor-rabbitmq pulls reactor 3.2.8 anyway